web-components/package.json

```diff
@@ -138,7 +138,7 @@
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.10.3",
-    "webpack-merge": "^4.2.2",
+    "webpack-merge": "^5.7.3",
```

Comment:
@Budhia What's up with the upgrade? Were you hitting issues with your build that should go library wide?

Reply:
@futurethang Yes. While I was trying to setup the repo the dist:dev command was giving issues related to the webpack version. So upgrading it solved the scenario.